benches/fibonacci.rs

```diff
@@ -7,7 +7,9 @@ use crate::helpers::*;
 
 #[bench]
 fn fib(bencher: &mut Bencher) {
-    bencher.iter(|| { fibonacci_helper::main(); })
+    bencher.iter(|| {
+        fibonacci_helper::main();
+    })
```

Comment:
The curly braces and semicolon are entirely unnecessary here, are they not?

Reply:
yes, but this is what rustfmt does

Reply:
"The machine told me do to this" is never a good argument though, that's what we have brains for. ;)